src/test/java/com/microsoft/azure/spring/data/documentdb/core/DocumentDbTemplateIT.java

```diff
@@ -141,4 +148,19 @@ public void testDeleteById() {
         assertThat(result.size()).isEqualTo(1);
         assertTrue(result.get(0).equals(person2));
     }
+
+
+    @Test
+    public void testDocumentDBAnnotation() {
+        final IndexingPolicy policy = collectionPerson.getIndexingPolicy();
+
+        Assert.notNull(collectionPerson, "class Person Collection should not be null");
```

Comment:
If we really need to add assertion check for collectionPerson here, and if the collectionPerson is null, it will fail in first line 155, the assertion will not run at all.

Reply:
yeah, make sense.